src/pepr/operator/bully.ts

```diff
@@ -0,0 +1,53 @@
+import { K8s, kind } from "pepr";
+import { UDSPackage } from "./crd";
+import { Direction, Gateway } from "./crd/generated/package-v1alpha1";
+
+const intervalMS = 50;
+let inc = 0;
+
+setInterval(async () => {
```

Comment:
After a couple minutes poking around, I'm not sure what this file is meant more, maybe add comments?

Reply:
yeah thanks, I talked about it the now archived slack thread 😆 I'll add something, it's just a temporary stress test I was running